src/pages/TeamPage/TeamPage.js

```diff
@@ -1,40 +1,178 @@
+      </div>
+
+      <div className="spacer-line"></div>
+      <div className="spacer"></div>
 
-                <img
-                    style={{ width: "100%", height: "auto" }}
-                    src="/TeamHeaderPhoto.png"
-                    alt="Header"
+      <div className="grid-title-group">
+        <div className="grid-title"> 
+          <p>CWRU MedWish Team</p>
+        </div>
+        <div className="grid-title-line"></div>
+      </div>
+
+      <div className="images-title-spacer"></div>
+      <div className="spacer"></div>
+
+      <Card>
+        <CardContent>
+          <Typography variant="h4">{}</Typography>
+          <Grid container spacing={2} className="card-container">
+            
+            {events.map((event) => (
+              <Grid item key={event.id} xs="auto" sm="auto" md="auto">
+                <TeamCard
+                  name={event.name}
+                  position={event.position}
+                  linkedin_link={event.linkedin_link}
+                  mailto_link={event.mailto_link}
+                  image_link={event.image_link}
                 />
+              </Grid>
+            ))}
+          </Grid>
+        </CardContent>
+      </Card>
+            
+      <div className="spacer"></div>
+      <div className="spacer-line"> </div>
+      <div className="spacer"></div>
 
-                <div className="team-text">
-                <h1>Team</h1>
-                </div>
-            </header> 
+      
+        <div className="contact-title"> 
+          <p>Contact Us</p>
+        </div>
 
-            <div className="spacer"></div>
+      <div className="email-link">
+        <a href={`mailto:${contactUs_link}`}>cwrumedwish-exec@case.edu</a>
+      </div>
 
-            <div className="spacer-line"> </div>
+      <div className="spacer"></div>
+      <div className="spacer"></div>
+      
+      <div class="contact-form">
+        <form ref={form} onSubmit={sendEmail}>
+          <div class="form-row">
+            <div class="form-group">
+              <label for="firstName">First Name</label>
+              <input type="text" id="firstName" name="firstName" required></input>
+            </div>
                 
-            <div className="horizontal-center-text">
-                <p>CWRU MedWish is empowered by its executive board, faculty advisors, MedWish International mentors, and all of its members! Check out our team below.</p>
+            <div class="form-group">
+              <label for="lastName">Last Name</label>
+              <input type="text" id="lastName" name="lastName" required></input>
             </div>
+          </div>
 
-            <div className="spacer-line"> </div>
-            
-            <div className="spacer"></div>
+          <div className="spacer"></div>
 
-        </div>
-    ); 
+          <div class="form-row">
+            <div class="form-group">
+              <label for="email">Email Id</label>
```

Comment:
```suggestion
              <label for="email">Email ID</label>
```

I had this wrong in the original figma design!